### DISCUSSION TITLE
How to apply custom style to FluentUI Web Components

### DISCUSSION THREAD

**Discussion by da-ms-22** [microsoft/fluentui-blazor#232] [Q&A] [answered]
Hi, I'm using Dialog component in a Blazor wasm project. The default dialog is rendered in an unreadable way, as shown in below screenshot.
I want to adjust the margin top, width and height of the dialog. I've tried various CSS methods. And there is no built in design token for margin, height or width. How to style it?

Code:
![image](https://user-images.githubusercontent.com/114713693/193348773-0c28f8d8-8d62-449c-a524-0aea2c7eb5f4.png)

In browser:
![image](https://user-images.githubusercontent.com/114713693/193348792-dbce06b8-01cd-4d3c-aca9-de6ee52c4fc6.png)

**Comment by vnbaaij** [comment, marked as answer]
Hi,
Regarding the height and width, see https://github.com/microsoft/fast-blazor/issues/207. You can address margin through css as well with something like:
```
::deep > fluent-dialog::part(control) {
    --dialog-width: 300px;
    --dialog-height: 350px;
    padding: 2rem;
}
```
Beware that the `<FluentDialog>` needs to be in a `<div>` or something like that for the `::deep` to work.

**Reply by da-ms-22** [reply]
Thank you very much Vincent! This is great help to our project. Also the control now it has its dedicated `<div>`!
